src/components/SidePane/getHelpContent.tsx

```diff
@@ -20,50 +23,337 @@ const helpContentMap: Record<string, HelpContent> = {
     r: {
         content: (styles: ThemeStyles) => (
             <>
-                <Text style={[styles.textHeadlineH1, styles.mb4]}>Chat Reports</Text>
-                <Text style={styles.textNormal}>... general chat reports help ...</Text>
+                <Text style={[styles.textHeadlineH1, styles.mb4]}>Inbox</Text>
+                <Text style={styles.textNormal}>
+                    Chat is the foundation of New Expensify. Every expense, expense report, workspace, or member has an associated "chat", which you can use to record additional details, or
+                    collaborate with others. Every chat has the following components:
+                </Text>
+
+                <Text style={[styles.textHeadlineH2, styles.mv4]}>Header</Text>
+                <Text style={styles.textNormal}>
+                    This shows who you are chatting with (or what you are chatting about). You can press the header for more details on the chat, or additional actions to take upon it.
+                </Text>
+
+                <Text style={[styles.textHeadlineH2, styles.mv4]}>Comments</Text>
+                <Text style={styles.textNormal}>The core of the chat are its comments, which come in many forms:</Text>
+                <Text style={[styles.textNormal, styles.mt2]}>
+                    {CONST.BULLET} <Text style={styles.textBold}>Text</Text> - Rich text messages stored securely and delivered via web, app, email, or SMS.
```

Comment:
Out of curiosity, why are we using `CONST.BULLET` for these and not a regular `<ul></ul>`?

Reply:
That is a great question! There is no equivalent of `ul`/`li` in React Native, so instead devs just use unicode characters for this. [Source](https://stackoverflow.com/questions/39110460/react-native-unordered-style-list)